datumaro/plugins/coco_format/converter.py

```diff
@@ -605,6 +607,12 @@ def patch(cls, dataset, patch, save_dir, **kwargs):
             if osp.isfile(image_path):
                 os.unlink(image_path)
 
+            image_path = osp.join(images_dir, subset,
+                conv._make_image_filename(item))
+            if osp.isfile(image_path):
+                os.unlink(image_path)
```

Comment:
The same code is above

Reply:
It mostly is, but the second block has `subset` part in the path. We need to check and remove both of the images.